packages/chain-adapters/src/evm/avalanche/AvalancheChainAdapter.test.ts

```diff
@@ -406,7 +406,7 @@ describe('AvalancheChainAdapter', () => {
       } as unknown as BuildSendTxInput<KnownChainIds.AvalancheMainnet>
 
       await expect(adapter.buildSendTransaction(tx)).rejects.toThrow(
-        'AvalancheChainAdapter: to is required',
+        'Avalanche C-Chain ChainAdapter: to is required',
```

Comment:
Use the `ChainAdapterName` enum in tests.

Reply:
f98799710837fde0a943a3e4e0e0259650dbb8f0